play/psycopg3/README.md

```diff
@@ -0,0 +1,6 @@
+# Example using TAIL with psycopg3
```

Comment:
I'd prefer to keep `psycopg3` in the directory name until it is the recommended version of psycopg, because until that time this should not be construed as the production-ready thing to use.

Reply:
yeah I'd be happy with `tail-python-psycopg3` or `stream-python-psycopg3`, it's just that `psycopg3` by itself is pretty in-the-weeds.

Reply:
Eh, those are pretty unwieldy. I’d prefer either `tail-psycopg3` or how it currently is (just `psycopg3`). These directory names don’t need to be thorough descriptions. “Psycopg3” is both pithy and unambiguous if you Google it. Plus there’s a README.